modules/mesh_com/entrypoint.sh

```diff
@@ -94,7 +86,5 @@ else
 
     # Start mesh publisher
     ros2 run mesh_com mesh_publisher --ros-args -r __ns:=/$DRONE_DEVICE_ID &
```

Comment:
Remove & in the end of the command to leave the ros node running and keep container alive.

Reply:
This is being addressed in another PR: https://github.com/tiiuae/mesh_com/pull/304/
Perhaps I should combine these both in to one?